src/pages/nft/claim/index.vue

```diff
@@ -739,15 +742,38 @@ export default {
   async mounted() {
     const { redirect, free, from, state, ...query } = this.$route.query;
     let price;
-    try {
-      const { data } = await this.$api.get(
-        getNFTBookPaymentStatusEndpoint({
-          classId: this.classId,
-          collectionId: this.collectionId,
-          paymentId: this.paymentId,
-        })
-      );
-      ({ price } = data);
+    if (this.paymentId) {
+      try {
+        const { data } = await this.$api.get(
+          getNFTBookPaymentStatusEndpoint({
+            classId: this.classId,
+            collectionId: this.collectionId,
+            paymentId: this.paymentId,
+          })
+        );
+        ({ price } = data);
+        const {
+          giftInfo,
+          isPhysicalOnly,
+          status,
```

Comment:
Why there is `giftInfo` and `status` in a class ID price data? What are their expected value?